src/Tizen.NUI/src/public/WebView/WebContext.cs

```diff
@@ -515,7 +515,7 @@ public void SetContextTimeZoneOffset(float offset, float time)
         /// Registers url schemes enabled.
```

Comment:
url -> URL

Reply:
Done.
I updated it in another patch https://github.com/Samsung/TizenFX/pull/3520